core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -135,6 +149,27 @@ object ConsumerGroupCommand extends Logging {
     }
   }
 
+  def printAssignmentResetted(groupAssignmentsResetted: Map[PartitionAssignmentState, Long], useNewConsumer: Boolean): Unit = {
```

Comment:
Do we really need a new method which does almost exactly the same thing as `printAssignment`? Perhaps we can just pass the new offsets in an optional parameter?